docs/Xdistro-OL8.md

```diff
@@ -0,0 +1,21 @@
+To support Oracle Linux 8, it will be necessary to install the `oracle-linux-release` and `oracle-linux-release-el8` RPMs. As of the writing of this document, these RPMs may be found at [`https://yum.oracle.com/oracle-linux-8.html`](https://yum.oracle.com/oracle-linux-8.html). The following links are correct as of this document's writing and are suitable for creating an Oracle Linux 8.9 AMI:
+
+- [oracle-linux-release](https://yum.oracle.com/repo/OracleLinux/OL8/baseos/latest/x86_64/getPackage/oraclelinux-release-8.9-1.0.8.el8.x86_64.rpm)
+- [oracle-linux-release-el8](https://yum.oracle.com/repo/OracleLinux/OL8/baseos/latest/x86_64/getPackage/oraclelinux-release-el8-1.0-35.el8.x86_64.rpm)
+
+If building an Oracle Linux 8.10 AMI or building in an environment where the public Oracle yum repositories are not available, it will be necessary to locate the correct RPM URLs for your build-needs.
+
+To download/stage these RPMs, execute:
+
+~~~bash
+XdistroSetup.sh -d Oracle \
+  -k <oraclelinux-release_RPM_URL> \
+  -r <oraclelinux-release_RPM_URL>,<oraclelinux-release-el8_RPM_URL> 
```

Comment:
Maybe use the actual URLs like you did with the CentOS8Stream instructions?

Reply:
Guidance was meant to be generalized since the "last known good" links were shared earlier in the file and the document was meant to provide generic guidance across environments. More a case that I need to generalize the CO8 guidance.

Updating the document &ndash; just realized it wasn't as complete as it needed to be.